corehq/apps/userreports/reports/specs.py

```diff
@@ -297,13 +297,23 @@ def get_display(self):
         return self.display or self.value
 
 
+def _validate_filter_slug(s):
+    if "-" in s:
+        raise Exception(_(
+            """
+            Filter slugs must be legal sqlalchemy bind parameter names.
+            '-' and other special character are prohibited
```

Comment:
Spent a while in the sqlalchemy docs but couldn't find any explanation of what characters are allowed or disallowed